docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<p>I think we should really set <code>k=10</code>  and compare the results with <code>stumpy.stump</code>  with <code>k=10</code>  as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Will be revised in he next commits.